deployment/console-backend-deploy.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: console-backend
+  labels:
+    app.kubernetes.io/component: console-backend
+    app.kubernetes.io/name: console-backend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: console-backend
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: console-backend
```

Comment:
Could you use all the labels that TAS is using for easy filtering?
https://github.com/securesign/secure-sign-operator/blob/main/config/samples/rhtas_v1alpha1_securesign.yaml#L4-L7?